SpotifyAPI/Web/SpotifyWebAPI.cs

```diff
@@ -44,6 +44,28 @@ public void Dispose()
             GC.SuppressFinalize(this);
         }
 
+        #region Configuration
+        /// <summary>
+        /// Specifies after how many miliseconds should a failed request be retried.
+        /// </summary>
+        public int RetryAfter { get; set; } = 50;
+
+        /// <summary>
+        /// Should a failed request (specified by <see cref="RetryErrorCodes"/> be automatically retried or not.
+        /// </summary>
+        public bool UseAutoRetry { get; set; } = false;
+
+        /// <summary>
+        /// Maximum number of tries for one failed request.
+        /// </summary>
+        public int RetryTimes { get; set; } = 10;
+
+        /// <summary>
+        /// Error codes that will trigger auto-retry if <see cref="UseAutoRetry"/> is enabled.
+        /// </summary>
+        public IEnumerable<int> RetryErrorCodes { get; private set; } = new int[] { 500, 502, 503 };
```

Comment:
Hmm, looking at it... not really. For some reason I thought it shouldn't be a part of public API but can't remember why right now.